tests/lib/rules/prefer-user-event.test.ts

```diff
@@ -295,10 +320,21 @@ ruleTester.run(RULE_NAME, rule, {
           endColumn: 22,
           data: {
             userEventMethods:
-              'userEvent.click(), userEvent.type() or userEvent.deselectOptions()',
+              'userEvent.click, userEvent.type, userEvent.selectOptions, or userEvent.deselectOptions',
```

Comment:
@epmatsw nice catch opening this issue! This test should have caught that, but has the wrong expected output :(